src/vsync.js

```diff
@@ -116,6 +127,14 @@ export class Vsync {
   }
 
   /**
+   * Runs the measure operation via vsync.
+   * @param {function()} mutator
```

Comment:
I mean the parameter "mutator". It should be renamed to make clear that it's for measuring.